assets/scss/layouts/products/_productSaleBadges.scss

```diff
@@ -64,6 +64,16 @@
     z-index: zIndex("high");
 }
 
+.product:hover .starwrap {
+    
+    .sale-flag-star,
+    .sale-flag-star:before,
+    .sale-flag-star:after {
+        background: stencilColor("color_hover_product_sale_badges");
+        transition: 800ms ease;        
```

Comment:
I recommend moving the `transition` property to the place where declaring the main styles of these selectors.

Also, you need to specify `transition-property`, the next example would be helpful

```
transition: background-color 800ms ease;
```

Reply:
Update scss file following through your comments. We have 3 types of sale badges as I mentioned earlier, so I applied such changes for other badges too